server/src/port/persistence/repository/ghost.repository.ts

```diff
@@ -0,0 +1,12 @@
+import { DeepPartial, EntityRepository, Repository, SaveOptions } from 'typeorm';
+import { Ghost } from '../../../domain/ghost/ghost.entity';
+
+@EntityRepository(Ghost)
+export class GhostRepository extends Repository<Ghost> implements IGhostRepository {
+
+}
+
+interface IGhostRepository {
```

Comment:
Please divide the interface `IGhostRepository` and the implementation `GhostRepository` in `ghost.repository.ts`.